hosting/src/firebase.ts

```diff
@@ -0,0 +1,25 @@
+import { initializeApp } from "firebase/app";
+import { connectAuthEmulator, getAuth } from "firebase/auth";
+import { connectFirestoreEmulator, getFirestore } from "firebase/firestore";
+
+export const app = initializeApp({
+	apiKey: "AIzaSyBZh6KFSTk8Ue33Lve7CETgJU9JpTfBAJg",
```

Comment:
Would we want to env var these? Be useful if developing with a test firebase instance

Reply:
The emulator automatically overwrites these values so I don't see a need personally